straitlets/ext/click.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import absolute_import
+
+from operator import itemgetter
+
+import click
```

Comment:
Should we do any special handling here if the user doesn't have `click` installed?